src/rshim.c

```diff
@@ -1989,6 +2028,82 @@ static void rshim_boot_workaround_check(rshim_backend_t *bd)
   }
 }
 
+static int rshim_bf2_a0_wa(rshim_backend_t *bd)
+{
+  int i, rc;
```

Comment:
Maybe move this line after line 2035?  The code tries to follow the 'mellanox' way to sort the lines in descending according to the line width.